Hymns/Common/WebView.swift

```diff
@@ -4,15 +4,36 @@ import WebKit
 /**
  * Generic container to display a URL wiithin the app.
  */
+
 struct WebView: UIViewRepresentable {
 
-    let request: URLRequest
+    static var cache = [URL: WKWebView]()
+
+    let url: URL?
 
     func makeUIView(context: Context) -> WKWebView {
-        return WKWebView()
+        guard let url = url else {
+            // TODO show error state
+             fatalError()
```

Comment:
What does fatalError do? I don't think we want to crash the app if the url is empty... Maybe just showing an empty/error state is good enough. (maybe we can ask Emily to design one)

Reply:
Good point... Yeah that would be good.